scripts/simex

```diff
@@ -253,6 +253,37 @@ builds_make_parser.set_defaults(cmd=do_builds_make)
 
 # ---------------------------------------------------------------------------------------
 
+def do_dev_builds(args):
+	return do_dev_builds_make(args)
+
+dev_builds_parser = main_subcmds.add_parser('develop', help='Build local programs',
+		aliases=['d'], parents=[phase_selection_parser])
+dev_builds_parser.set_defaults(cmd=do_dev_builds)
+dev_builds_parser.add_argument('--revision', type=str)
+dev_builds_parser.add_argument('build', type=str)
```

Comment:
We could support multiple builds on the command line.

Reply:
I renamed the `build` argument to `builds` and added the parameter `nargs='+'` to the argument. You can now use the command `simex develop <build1> <build2> <build ...>` to apply actions on `build1`, `build2`, ...